.github/workflows/release-nightly.yml

```diff
@@ -20,19 +20,19 @@ jobs:
         uses: actions/checkout@v4
       - id: latest-commit-hash
         name: Print latest commit
-        run: echo $(git rev-parse HEAD) && echo "::set-output name=latest-commit-hash::$(git rev-parse HEAD)"
+        run: echo $(git rev-parse HEAD) && echo "latest-commit-hash=$(git rev-parse HEAD)" >> $GITHUB_OUTPUT
```

Comment:
this syntax was deprecated and triggered a warning.
replaced with a [recommended approach](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/).